server/database/models/tournamentModel.ts

```diff
@@ -1,2 +1,40 @@
-// import mongoose from 'mongoose';
-// import Tournament from './../../../shared/types/Tournament';
+import mongoose, { Schema, Document } from "mongoose";
+import { normalize } from "node:path";
+import userRouter from "../../routes/api/userRoutes";
+import Tournament from "./../../../shared/types/Tournament";
+import User from "./../../../shared/types/User";
+
+// https://github.com/Appsilon/styleguide/wiki/mongoose-typescript-models
+// https://medium.com/@agentwhs/complete-guide-for-typescript-for-mongoose-for-node-js-8cc0a7e470c1
+// https://mongoosejs.com/docs/subdocs.html
+
+const UserSchema = new Schema({
+  id: String,
```

Comment:
Myślałem o tym później i te konflikty o których Przemek mówiłeś wynikają z tego ze mongo tworzy zarówno _id jak i id (zauważyłem, że nie zawsze jest fizycznie w bazie ALE zawsze mozna się odwołać do _id poprzez id). Co ciekawe są one różnego typu co sprawia czasem problemy. Dlatego uważam, że to pole powinno nazywać się inaczej np. po prostu user albo user_microsoft_id

Reply:
I've talked to him about this, I think `microsoftId` sound good. I suggest adding length restrictions for e.g. strings, array.
Use English in PR :D